src/do-deploy.js

```diff
@@ -3,6 +3,7 @@ const specifiedSiteInPr = require('./helpers/specified-site-in-pr');
 const postCommentOnGithub = require('./helpers/post-comment-on-github');
 const updatePrDescriptionWithLinks = require('./helpers/update-pr-description-with-links');
 const writeEnvVariablesFile = require('./helpers/write-env-variables-file');
+const writeEnvVariablesToProcess = require('./helpers/write-env-variables-to-process');
```

Comment:
I think it can be made to be a bit more descriptive, what do you think about `exposeEnvVarsFromPrToProcess`?

Reply:
Done, with a slight modification on the suggested name. Let me know if you want it changed 🙂